java/bin/start.sh

```diff
@@ -36,13 +58,31 @@ if [ -z "$SKIP_BUILD" ]; then
   #Change PWD to root folder for Maven Build
   cd ${PROJECT_ROOT_DIR}
   mvn clean spotless:apply install -DskipTests
+  build_status=$?
+
+  if test $build_status -eq 0
+  then
+          printf "\n Code build went successful, thus we are good to go \n"
+  else
+          printf "\n We ran into some issues while buiding the jar file, seems like mvn clean install is not running fine \n"
```

Comment:
If build is not success you need to stop the execution on this script.
Otherwise there is no value in checking for it.

Reply:
Correspondingly made changes in the code.